src/Components/Row/index.js

```diff
@@ -40,8 +40,15 @@ export default {
     {
       id: generateUUID(),
       type: "Row",
-      content: [],
-      attrs: defaultAttrs(),
+      content: [
```

Comment:
I think this always wraps every newly added Single Column Row in another row, which we probably don't want